src/monodroid/jni/dylib-mono.h

```diff
@@ -109,6 +109,36 @@ typedef struct {
 	const unsigned int size;
 } MonoBundledAssembly;
 
+// Keep this in sync with MonoObject in object.h
+typedef struct {
+	// MonoObject 2 pointers
+	void *_pad_mono_object[2];
+} MonoObject;
+
+// Keep this in sync with MonoAppDomainSetup in domains-internal.h
+typedef struct {
+	MonoObject object;
+	// MonoAppDomainSetup unused entries
+	void *_pad_mono_appdomainsetup[9];
+
+	MonoString *shadow_copy_files;
```

Comment:
This, at least, requires the struct declaration, but I wish there were a way that didn't require having this declaration.